### ISSUE TITLE
Discussion: Combat scaling

### ISSUE THREAD

**Issue by Zironic** [Arx-Game/arxcode#67]
Currently Arx uses a relatively flat damage mitigation calculation, the same kind of damage reduction you see in a lot of table top RPG's because it makes things easy to calculate, Armor-damage= Damage taken, easy.

This kind of damage calculation isn't quite as common in computer games and the reason for that is because it creates a very twisted engagement profile where low end threats deal no damage at all, mid range threats are all over the place and high end threats/crits just blast through armor as if it wasn't there.

What's pretty common because it works very well is a model where you decide that 1 armor should be worth X% of effective health. Then mitigation simply becomes 1 / (1+x*armor). For instance at x=5% and armor values at 30 and 60 you get.  1 / (1+0,05*30) = 0.4% (effectively 150% bonus hp) and 1 / (1+0,05*60)  = 0.25% (effectively 300% bonus hp)

**Comment by TehomCD** [comment]
Yup, I've wanted to change it to a mitigation percentage at some point. Probably worth doing at the same time that mitigation becomes specific to damage-type/attack-type, what have you.
